pkg/api/validation/validation.go

```diff
@@ -444,6 +444,10 @@ func validateVolumeSource(source *api.VolumeSource, fldPath *field.Path) field.E
 			numVolumes++
 			allErrs = append(allErrs, validateISCSIVolumeSource(source.ISCSI, fldPath.Child("iscsi"))...)
 		}
```

Comment:
passing unbounded data through to native exec calls run as root always makes me squeamish... I wasn't sure how well the iscsiadm tool handled malicious data